ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -63,6 +64,20 @@ export default function EditGasPopover({
 
   const minimumGasLimitDec = hexToDecimal(minimumGasLimit);
 
+  const updatedCustomGasSettings = useIncrementedGasFees(transaction);
+
+  let transactionToUse = transaction;
```

Comment:
Nit: This name is a bit confusing :thinking: Maybe something like `customizedTransaction` would be better? Or `updatedTransaction`? :shrug:

Reply:
I'll roll with updateTransaction.